### ISSUE TITLE
basic_multi_auth not working with encoded password in configuration file

### ISSUE THREAD

**Issue by mzico** [GluuFederation/oxAuth#1409]
If we use encoded password in `multi_auth_conf.json` file, script can't connect to backend servers. 
Clear text password  in 'bindPassword' attirbute in this configuration file ( multi_auth_conf.json ), working okay. 

Not working condition screenshot below: 

![Capture](https://user-images.githubusercontent.com/7156097/86462866-fca47d00-bcf1-11ea-9a59-04458256d0f2.PNG)

oxAuth and identity version below: 

```
[root@test42 temp]# cat jetty-localhost-8081-oxauth_war-_oxauth-any-11086820290422049047.dir/webapp/META-INF/MANIFEST.MF
Manifest-Version: 1.0
Implementation-Title: oxAuth Server
Build-Branch: origin/master
Implementation-Version: 4.2.0-SNAPSHOT
Archiver-Version: Plexus Archiver
Built-By: jetty
Implementation-Build: 4f32bd5f69eda6abd8115556a221f8475e530927
Implementation-Vendor-Id: org.gluu
Created-By: Apache Maven 3.3.9
Build-Jdk: 1.8.0_221

[root@test42 temp]# cat jetty-localhost-8082-identity_war-_identity-any-10395835038213216889.dir/webapp/META-INF/MANIFEST.MF
Manifest-Version: 1.0
Implementation-Title: oxTrust Server
Build-Branch: origin/master
Implementation-Version: 4.2.0-SNAPSHOT
Archiver-Version: Plexus Archiver
Built-By: jetty
Implementation-Build: 36dd824d8878d1f17aa50a62d907afee56410d14
Implementation-Vendor-Id: org.gluu
Created-By: Apache Maven 3.3.9
Build-Jdk: 1.8.0_222

[root@test42 temp]#
```

**Comment by mzico** [comment]
@yurem : any thought on this?

**Comment by yuriyz** [comment]
I will check.

**Comment by yuriyz** [comment]
@mzico you are right, clear password was used by script. Script is fixed in 4.2 (decryption is performed).

https://github.com/GluuFederation/oxAuth/blob/357d9ab02cc3b4a61f4dd215e37aa220c7af27e5/Server/integrations/basic.multi_auth_conf/BasicMultiAuthConfExternalAuthenticator.py#L261
